manifests/r/Ryochan7/DS4Windows/3.3.0/Ryochan7.DS4Windows.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Ryochan7.DS4Windows
+PackageVersion: 3.3.0
+InstallerType: nullsoft
+Scope: machine
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
+  - PackageIdentifier: ViGEm.ViGEmBus
+  - PackageIdentifier: ViGEm.HidHide
```

Comment:
```suggestion
```

Based on https://github.com/Ryochan7/DS4Windows#requirements, I don't think these two dependencies are necessary. Can you remove them and then we can check if the package passes the validation pipelines or not. If it doesn't, we can add them back in. If it does, we should also remove these dependencies in the previous version.

Reply:
Nope. Manually validated in Windows Sandbox, the DS4Windows installer will automatically launch wizard of `ViGEm.ViGemBus` and `ViGEm.HidHide`. This behaviour can not be bypassed during installation.

As the above 2 packages are already presented in the repository and can be installed unattendedly by `winget.exe`, we HAVE TO add them as package dependencies, preventing their installation wizard from launching by DS4Windows'.

![image](https://github.com/microsoft/winget-pkgs/assets/49941141/c8155984-0e5c-49f9-844a-00afc665d3ed)

![image](https://github.com/microsoft/winget-pkgs/assets/49941141/f5cd51d6-2705-4ca5-8ffc-c0a561d7c6d6)